### DISCUSSION TITLE
Error message on install

### DISCUSSION THREAD

**Discussion by FloppyDisco** [synek/git-plan#104] [Help & Questions]
Hi, 
I'm getting an error message when running git plan on my machine
I installed it with pip, ran `git plan init` in  a test repository
and now when I run any `git plan` commands i get an error saying 
```
RuntimeError: Couldn't find an editor installed on your system.
```
not really sure what this is looking for, but just wondering if you had any ideas.

**Comment by rorybyrne** [comment]
Thanks for the issue.

Looks like it's coming from [here](https://github.com/rorybyrne/git-plan/blob/c4750f39db5321280816aeecc086642fcb627196/git_plan/service/plan.py#L89). When `git plan` runs, it tries to open an editor on your terminal so you can type in a plan, but it couldn't find any installed text editor.

What is the output of `echo $EDITOR`? Do you have `vim` installed? What text editor do you normally use, e.g. when you run `git commit` what editor is opened?

**Reply by FloppyDisco** [reply]
ah. i see. 
hmm. 
i tried setting `EDITOR` in my `.zshrc` 
but it does not seem to work. 

<img width="1015" alt="Screenshot 2024-12-12 at 6 00 36 PM" src="https://github.com/user-attachments/assets/87ca53bd-4a2e-4ad2-8c29-d2b17d2c5e67" />
<img width="1019" alt="Screenshot 2024-12-12 at 6 03 42 PM" src="https://github.com/user-attachments/assets/7f1b1882-a412-422b-b011-f405fed8692b" />

**Reply by rorybyrne** [reply]
Ok I'll check it tomorrow under these conditions! Thanks for reporting.

**Comment by rorybyrne** [comment]
You may have to 'source ~/.zshrc' after that. What does 'echo $EDITOR' say?

**Comment by FloppyDisco** [comment]
it's in the images. 
`EDITOR` was not previously set. but I tested with both 'code' and 'nano' and neither worked.
